src/daidepp/keywords.py

```diff
@@ -344,33 +354,33 @@ def __str__(self):
 class CHO:
     start_year: int
```

Comment:
Isn't this supposed to be minimum and maximum? Not start year and end year

Reply:
changed to `minimum` and `maximum`